src/kubectl-object-model.ts

```diff
@@ -137,4 +152,31 @@ export class Kubectl {
         if (typeof str === 'string') { return str; }
         return str.join(',');
     }
+
+    private populateDeployedObjects(configurationPaths: string | string[]) {

```

Comment:
Move this to `files-helper`.

Reply:
This looks like a kubectl related operation rather than a generic file related one. Should we consider moving it to kubectl-utils then? I think the latter looks better.